plugins/org.preesm.codegen.xtend/src/org/preesm/codegen/xtend/spider/visitor/SpiderCodegen.java

```diff
@@ -942,6 +978,7 @@ private void generateFunctionBody(final AbstractActor aa) {
         switch (param.getDirection()) {
           case IN:
             if (param.isIsConfigurationParameter()) {
+              aa.isLocallyStatic();
```

Comment:
this line does nothing if the result is not used

Reply:
Removed